README.md

```diff
@@ -34,29 +34,25 @@
 ```jsx
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { createStore } from 'redux';
 import { createInputsReducer, connectWithInputs, ReduxInputsWrapper } from 'redux-inputs';
 import { Provider } from 'react-redux';
-import thunk = from 'redux-thunk';
```

Comment:
Shoot, and it appears the thunk is required. Can we not require this unless we are using async validators??